app/controllers/general.php

```diff
@@ -289,19 +291,74 @@
         ->noContent();
 }, ['request', 'response']);
 
-App::error(function ($error, $utopia, $request, $response, $layout, $project) {
+App::error(function ($error, $utopia, $request, $response, $layout, $project, $logger, $loggerBreadcrumbs) {
     /** @var Exception $error */
     /** @var Utopia\App $utopia */
     /** @var Utopia\Swoole\Request $request */
     /** @var Appwrite\Utopia\Response $response */
     /** @var Utopia\View $layout */
     /** @var Appwrite\Database\Document $project */
+    /** @var Utopia\Logger\Logger $logger */
+    /** @var Utopia\Logger\Log\Breadcrumb[] $loggerBreadcrumbs */
+
+    $user = null;
+    /** @var Appwrite\Database\Document $user */
+
+    try {
+        $user = $utopia->getResource('user');
+    } catch(\Throwable $th) {
+        // All good, user is optional information for logger
+    }
```

Comment:
This should only be done if logger is used right?

Reply:
Correct, moved it.